dcos/dcos-ui
# DCOS-44594: COPS-3759 affecting dcos-ui: When no nodes are in the cluster the message should have different wording

Changed the message when there are no nodes to be more accurate.

Closes https://jira.mesosphere.com/browse/DCOS-44594

## Testing
1. In `plugins/nodes/src/js/pages/NodesAgents.js` change line 314 to be `if (true) {`
2. Click on Nodes tab.
3. Verify that the message shown is changed.

## Trade-offs
Also changed "a" to "are".

## Dependencies
None.

## Screenshots

### Before
![screenshot from 2018-12-07 10-51-54](https://user-images.githubusercontent.com/40791275/49637839-324a8c80-fa0f-11e8-9169-37cb678d59a2.png)

### After
![screenshot from 2018-12-07 10-51-18](https://user-images.githubusercontent.com/40791275/49637845-35de1380-fa0f-11e8-9443-0edbabdea368.png)
